pkg/eraser/eraser.go

```diff
@@ -229,8 +263,11 @@ func removeVulnerableImages(c Client, socketPath string, imagelistName string) (
 		allImages2 = append(allImages2, img.Id)
 	}
 
-	log.Println("\nAll images following remove: ")
+	log.Println("\n\nAll images following remove: ")
```

Comment:
let's move the logs to a single line json. we want them to be parsable if a user wants to

Reply:
we can use https://github.com/go-logr/zapr. this can be a follow up. please create an issue if so